src/opencast.js

```diff
@@ -86,9 +90,47 @@ export class Opencast {
 
     await catchRequestError(async () => await self.updateUser());
 
+    await self.handleSeriesIDSelection();
```

Comment:
Why not put that into the `catchRequestError` arrow function above?

Reply:
Havent thought of that. Sounds like a good idea. :) I'll put in there.